packages/flutter/lib/src/material/data_table.dart

```diff
@@ -179,6 +184,8 @@ class DataRow {
   /// table.
   final List<DataCell> cells;
 
+  /// {@macro flutter.material.dataColumn.heading}
+  /// Summarizes the information displayed in this [DataRow].
```

Comment:
Information is lacking, for example that this will be announced by screenreaders,

Reply:
Thanks! Done.